lib/perl/Genome/Site/TGI/Extension/generate_lims_disk_unarchive.pl

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env lims-perl
+
+use strict;
+use warnings;
+
+use Data::Dumper;
+use Carp qw/confess/;
+
+use GSCApp;
+
+my $ii = GSC::IndexIllumina->get(analysis_id => $ARGV[0]);
```

Comment:
Added a check for `$ARGV[0]` and `[1]` to be defined.  Normally the user won't call this script directly, so I'm not fussed about making it too fancy!